dts/bindings/pinctrl/renesas,ra-pinctrl.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2023 TOKITA Hiroshi <tokita.hiroshi@fujitsu.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Renesas RA series pin controller
+
+compatible: "renesas,ra-pinctrl"
+
+include: base.yaml
+
+child-binding:
+  description: |
+    Definitions for a pinctrl state.
+  child-binding:
+
+    include:
+      - name: pincfg-node.yaml
```

Comment:
this should be removed entirely, none of the props are used right?

Reply:
I had a little misunderstanding. I deleted it additionally.
Thank you for pointing it out.